src/pages/registration/references_tab/components/DoiField.tsx

```diff
@@ -32,6 +32,7 @@ const DoiField = () => {
   return doiUrl ? (
     <>
       <TextField
+        data-testid="doi-field"
```

Comment:
tips: hvis du bare vil ha tak i selve `<input>` taggen er det enklere å bruke `inputProps={{ 'data-testid': 'message-input' }}`.
Men mulig det da blir verre å få tak i ClearIcon etc

Reply:
Det er ofte lettere og mer oversiktlig å gå  fra det overordende elementet ved å bruke `within()` istedet for å legge på flere data-testid for alle elementene eller bruke `parent()`